common/app/views/fragments/atoms/interactive.scala.html

```diff
@@ -0,0 +1,44 @@
+@(interactive: _root_.model.content.InteractiveAtom, shouldFence: Boolean)
+
+@body = {
+    <style>
+        @HtmlFormat.raw(interactive.css)
+    </style>
+    @HtmlFormat.raw(interactive.html)
+}
+
+@iframeBody = {
+    @body
+    <script>
```

Comment:
Can we abstract these scripts out into an interactives folder in JS files and use the `@InlineJs` helper. That way we can still integrate it into the build.